app/main/models/employee.py

```diff
@@ -0,0 +1,36 @@
+from app.main.db import db
+
+class Employee(db.Model):
```

Comment:
For db field names, let's start with lowercase to keep things consistent.

So:

```text
ID => id
FirstName => firstName
LastName => lastName
Email => email
```

Does this create DB table column names? If so, maybe it's better to go snake case? Thoughts?

Reply:
According to the [docs](https://docs.sqlalchemy.org/en/13/orm/mapping_columns.html), "A mapping by default shares the same name for a ```Column``` as that of the mapped attribute... The name assigned to the Python attribute which maps to ```Column``` can be different from either ```Column.name``` or ```Column.key``` just by assigning it that way".

Example:

```python
class User(Base):
    __tablename__ = 'user'
    id = Column('user_id', Integer, primary_key=True)
    name = Column('user_name', String(50))
```

So if I understand correctly, I think we can use snake case for the python attribute, then name it using whatever convention we want by passing it as the first argument of ```Column```.

I'm not sure what the convention is for SQL, but snake case seems easiest (at least it would be consistent with Python that way).